public/src/components/channelManagement/bannerTests/bannerTestVariantEditor.tsx

```diff
@@ -160,7 +160,7 @@ const BannerTestVariantContentEditor: React.FC<BannerTestVariantContentEditorPro
             <VariantEditorCopyLengthWarning charLimit={HEADER_COPY_RECOMMENDED_LENGTH} />
           )}
         </div>
-        )
+
```

Comment:
Not related, but I thought I'd clean this up!

Reply:
I'm not sure how this worked?!

Reply:
It was just in the markup! It meant in the middle of the variant editor there was a rogue `)` on it's own line between a couple of the fields!